src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health) {
+    this.name = name;
+    this.health = health;
```

Comment:
The `health` property should have a default value of 100 according to the task description. Consider setting the default value in the constructor of the `Animal` class.